src/backend/impl/src/repositories/types/user_profile.rs

```diff
@@ -24,6 +25,12 @@ pub struct UserProfile {
     pub config: UserConfig,
 }
 
+#[derive(Debug, CandidType, Deserialize, Clone, PartialEq, Eq)]
+pub struct SocialLink {
+    pub platform: String,
```

Comment:
Shouldn't the `platform` rather be an enum to keep things in order? The frontend could also use this enum to display the form fields properly

Reply:
I've switched this to be an enum now